File: Assignment 2/problem2.c
// Accept one number from user and print that number of * on screen

#include <stdio.h>
// void Display(int iValue)
// {
//     int iCunt = 0;
//     while (iCunt < iValue)
//     {
//         printf("* ");
//         iCunt++;
//     }
// }

void Display(int iValue)
{
    
    while ( iValue>0)
    {
        printf("* ");
        iValue--;
    }
}

int main()
{
    int iNo;
    printf("Enter number : ");
    scanf("%d", &iNo);

    Display(iNo);

    return 0;
}

File: Assignment 11/Problem2.java
//Accept N number from user and display smallest number from that.

/*
  Input : N = 6
          Elements = 85  66  96  45  29  88
          
  Output : 96
*/

import java.util.Scanner;

public class Problem2 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter N : ");
        int N = sc.nextInt();

        int[] arr = new int[N];
        System.out.println("Enter " + N + " elements : ");

        for (int i = 0; i < arr.length; i++) {
            arr[i] = sc.nextInt();
        }

        System.out.print("Output : ");
        int smallest = arr[0];
        for (int i = 0; i < arr.length; i++) {
            if (arr[i] < smallest) {
                smallest = arr[i];
            }
        }
        System.out.println(smallest);
    }

}
